packages/dx-react-demos/src/demo-sources/grid-basic/table-template.jsxt

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';<%&additionalImports%>
+import {
+  Grid,
+  Table,
+  TableHeaderRow,
+} from '@devexpress/dx-react-grid-<%&themeName%>';
+
+import {
+  generateRows,
+  globalSalesValues,
+} from '../../../demo-data/generator';
+
+/*
+.table-striped tbody tr:nth-of-type(odd) {
+  background-color: rgba(0,0,0,.05);
+}
+*/
+
+const TableComponent = ({ ...restProps }) => (
```

Comment:
Just a note. I don't like to add obvious postfix (e.g. xxxComponent for React components)